app/views/actual_db_schema/migrations/index.html.erb

```diff
@@ -2,7 +2,9 @@
 <html>
   <head>
     <title>Migrations</title>
-    <%= stylesheet_link_tag 'actual_db_schema/styles', media: 'all' %>
+    <%= csrf_meta_tags %>
```

Comment:
What's the point in using csrf_meta_tags here? Do we really need it taking into account that these forms will be used only in development mode?